src/Serilog.Sinks.ColoredConsole/Sinks/ColoredConsole/ColoredConsoleSink.cs

```diff
@@ -91,7 +91,12 @@ public void Emit(LogEvent logEvent)
                         {
                             RenderOutputToken(palette, outputToken, outputProperties, output);
                         }
-                        else switch (propertyToken.PropertyName)
+                        else
+                        {
+                            LogEventPropertyValue propertyValue;
+                            if (!outputProperties.TryGetValue(propertyToken.PropertyName, out propertyValue))
+                                continue;
```

Comment:
Yes, or even inverting the if to get rid of the continue. I'll try both and see which one looks the pretties :wink: